app/views/account_sign_up/new.html.erb

```diff
@@ -8,10 +8,9 @@
         <% if @account.errors.any? %>
           <div id="error_explanation" class="alert alert-danger">
             <%= pluralize(@account.errors.count, "error") %> prohibited this repository from being saved:
-
             <ul>
-            <% @account.errors.full_messages.each do |message| %>
-              <li><%= message %></li>
+            <% @account.errors.messages.each do |key, messages| %>
```

Comment:
The underlying ticket complained about identifying the field(s) in question when the response includes validation errors.  This is because the same model validation messages are triggered from different forms/controllers, but the forms do not expose the same fields.  

This step uses greater detail available from `errors` to include the rejected value in the message.  Exposing the `key` would be prerequisite to (for example) mapping the error message for one field (e.g. `cname`) back to a message referring to `Short Name`, where only the latter is exposed on the form.